### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by apedicdev** [breezefront/community#309] [1. Help] [answered]
Hi, 

is there a way to have a full dark theme? From the guide, I understand atm is only possible to have header and footer, but I might be wrong.

**Comment by vovayatsyuk** [comment, marked as answer]
Breeze Blank has dark theme support when the visitor setting is set to a prefer-dark color scheme. You can use it together with `@media-dark` hack to get the dark theme for all visitors:

```less
@respect-color-scheme: true;
// Trick to enable dark scheme for all visitors:
@media-dark: ~"(prefers-color-scheme: light), (prefers-color-scheme: dark)";
```

<details>
<summary>View result</summary>
<img width="600" src="https://github.com/user-attachments/assets/f46e53fa-32eb-485b-9a09-14bec8297163" alt="Dark Breeze Blank Theme"/>
</details>

As you can see from the [source code](https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/web/css/base/_variables.less#L92-L93) dark colors defined as CSS properties, so you can achieve the same effect without `(prefers-color-scheme: light), (prefers-color-scheme: dark)` trick:

```less
@base__text-color: @white
@base__background-color: @black;
// put other color variables here...
```

**Reply by apedicdev** [reply]
thank you so much!
